src/connection/GaugeListener.ts

```diff
@@ -3,19 +3,30 @@ import { MessageProcessorFactory } from "../processors/MessageProcessorFactory";
 import { GaugeConnection } from "./GaugeConnection";
 
 export class GaugeListener {
-    private readonly _processorFactory: MessageProcessorFactory;
-    constructor(factory: MessageProcessorFactory) {
-        this._processorFactory = factory;
-    }
 
-    public pollForMessages() {
-        let connection = new GaugeConnection();
-        this._processorFactory.on('messageProcessed', (message: gauge.messages.IMessage) => {
-            connection.write(message);
-        });
-        connection.on('messageReceived', async (message: gauge.messages.IMessage) => {
-            await this._processorFactory.process(message);
-        });
-        connection.start();
-    }
+  private readonly _processorFactory: MessageProcessorFactory;
+
+  constructor(factory: MessageProcessorFactory) {
+    this._processorFactory = factory;
+  }
+
+  public pollForMessages(): void {
+    const connection = new GaugeConnection();
```

Comment:
Line breaks after every variable definition can add unnecessary new lines.

Reply:
Lint is set to use line break after the whole set of variable definitions like set in here https://github.com/BugDiver/gauge-ts/pull/109/files/92c95fc1b93b15af4ba667061d96b1d4fe855e6d#diff-e4403a877d80de653400d88d85e4801aR16

Reply:
new lines, in this case, aren't unnecessary but expected to improve readability

Reply:
Just to be sure we are on the same page lint is expecting line break after this set of declarations

`
const v1 = 1;
const v2 = 1;
const v3 = 1;
let v4 = 1;

this.doSomethingSpecial();
`

So now you have a really clear vision of what's a declaration and what's part of the algorithm. We should definitely avoid mixing declarations and processing together even if would expect hoisting taking its place here.